bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/SurePetcareAPIHelper.java

```diff
@@ -78,9 +77,15 @@ public class SurePetcareAPIHelper {
     private String username = "";
     private String password = "";
 
+    private final String userAgent;
+
     private @NonNullByDefault({}) HttpClient httpClient;
     private SurePetcareTopology topologyCache = new SurePetcareTopology();
 
+    public SurePetcareAPIHelper() {
+        userAgent = "openHAB/" + FrameworkUtil.getBundle(this.getClass()).getVersion().toString();
```

Comment:
Thanks, I was not aware of this. It's probably cleaner to strip the build information, so the agent now will be "openHAB/4.0.0" where previously it would be e.g. "openHAB/4.0.0.202306011839".